src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.css

```diff
@@ -0,0 +1,35 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  margin-top: 15px;
```

Comment:
Margin to should be ~30

![Screenshot from 2021-04-20 13-55-06](https://user-images.githubusercontent.com/1193222/115387557-8cfd7100-a1e3-11eb-86a4-48b7673ff48a.png)